src/containers/AccountConnection.jsx

```diff
@@ -107,6 +106,8 @@ class AccountConnection extends Component {
 
         if (connection.error) {
           return Promise.reject(connection.error)
+        } else {
+          return Promise.resolve()
```

Comment:
We could resolve here with the connected account here instead of keep it empty, what do you think? We may use it later.